mnefun/_sss.py

```diff
@@ -1024,3 +1106,20 @@ def annotate_head_pos(raw, head_pos, rotation_limit=45, translation_limit=0.1,
 
     # Annotate on distance from the sensors
     return annot
+
+
+def check_sws():
```

Comment:
I see this new function creates a new instance of Params, which is fine because the sws parameters are populated from the config file in  Params.__init__().  But wouldn't the function have greater utility if it (optionally) accepted an existing 'params'?  Especially if a user for some reason has overwritten the parameter values after instantiation.  I tried this and it worked as expected: `check_sws(params)`

Reply:
The idea is that everyone should start using the new way of doing things (i.e., machine-based config without values set in the processing script), so *not* allowing an existing `Params` is actually preferred here.